mlflow/tracking/fluent.py

```diff
@@ -1978,8 +1967,8 @@ def get_autologging_params(autolog_fn):
 
     def setup_autologging(module):
         try:
-            autolog_fn = LIBRARY_TO_AUTOLOG_FN[module.__name__]
-
+            autologging_params = None
+            autolog_fn = eval(LIBRARY_TO_AUTOLOG_FN[module.__name__])
```

Comment:
Can we avoid using `eval` because it can lead to security vulnerabilities?

Reply:
Master code `autolog_fn = LIBRARY_TO_AUTOLOG_FN[module.__name__]` should work, what's the issue you want to address by adding `eval` ?

Reply:
the problem is `autolog()` imports all flavor modules, which is violating the lazy loading logic. 

`eval` is unsafe for inputs, but here it only takes in hardcoded string of module path.

Reply:
The issue with `eval` is that that makes us wonder "Is this really safe?". Can we use `importlib` instead?